http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/BaseCloudServiceRuntimeMonitor.scala

```diff
@@ -319,7 +319,8 @@ abstract class BaseCloudServiceRuntimeMonitor[F[_]] {
                                runtimeAndRuntimeConfig.runtime.runtimeName).map(b => (imageType, b))
           )
       }
-      availableTools <- streamFUntilDone(checkTools, 10, 5 seconds).compile.lastOrError
+      // wait for 10 minutes for tools to start up before time out.
+      availableTools <- streamFUntilDone(checkTools, 120, 5 seconds).compile.lastOrError
```

Comment:
If a tool needs that much time to come up, the runtime start time will be significantly affected, correct? Did we have some sort of alert for if that happens?

Reply:
👍 added metrics